### PR TITLE
Feat/cli audit

### PR DESCRIPTION
**Pull request by logan-anderson**
PR for https://github.com/tinacms/tinacms/issues/2356

I am going to scope down this PR to [this comment](https://github.com/tinacms/tinacms/issues/2356#issue-1078962687) plus. 
- [x] add docs for the `audit` command
- [x] make `label` optional

### CODE REVIEW

**Comment by logan-anderson** [comment]
Docs are here: https://github.com/tinacms/tinacms.org/pull/1155

**Review by jeffsee55 — APPROVED** [review]
This is looking good, next phase of this as discussed:

- [ ] Git checks to ensure that when `clean` is run, the user has a clean working directory
- [ ] Show extra fields found in markdown
- [ ] The error messages are GraphQL-specific, they should be more user friendly

File: packages/@tinacms/cli/README.md [hunk 1]
```diff
@@ -228,3 +231,8 @@ getPostsDocument(relativePath: "voteForPedro.md") {
 To learn how to work with this data on a Tina-enabled site, check out the [client documentation](https://tina.io/docs/tina-cloud/client/)
 
 > This API is currently somewhat limited. Specifically there's no support for filtering and sorting "list" queries. We have plans to tackle that in upcoming cycles
+
+
+## API Docs
```
[the remark this answers is not in the document]
Comment (logan-anderson) [reply packages/@tinacms/cli/README.md:236]: Yeah I did! I added a link to our API docs.

**Comment by logan-anderson** [comment]
> This is looking good, next phase of this as discussed:
> 
> * [ ]  Git checks to ensure that when `clean` is run, the user has a clean working directory
> * [ ]  Show extra fields found in markdown
> * [ ]  The error messages are GraphQL-specific, they should be more user friendly

@jeffsee55 I moved this into its own issue.

**Review by jeffsee55 — APPROVED** [review]

**Pull request merged**